share/shwap/data_id.go

```diff
@@ -17,7 +17,6 @@ const DataIDSize = RowIDSize + share.NamespaceSize
 
 // DataID is an unique identifier of a namespaced Data inside EDS Row.
 type DataID struct {
-	// TODO(@walldiss): why embed instead of just having a field?
```

Comment:
so you don't have to type .RowID to access its fields and allow passing DataID so where its RowID is expected. Also, we don't need to define GetHeight for every type and doing that on the new EdsID is enough